explainer.md

```diff
@@ -72,6 +72,113 @@ There are still many open questions, but a few big ones:
 - Will existing protocols ([example](https://github.com/openid/OpenID4VP/issues/125)) adopt this API?
 - Will regulation ([example](https://digital-strategy.ec.europa.eu/en/library/european-digital-identity-architecture-and-reference-framework-outline)) adopt this API?
 
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose,
+>      and for what purposes?
+
+The Digital Credential API exposes a one-time user-mediated communication channel from websites, to digital wallet applications, and back to websites with end-to-end encryption of the response. It is designed to be a better option than established lower-level communication channels like custom schemes, QR codes, and server-to-server network communication. How exactly this channel is used is up to the wallet applications and host operating system, but we are designing it to be suitable for conveying presentations of digital credentials such as claims in mobile driver’s licenses.
```

Comment:
Do we mean "end-to-end encryption" here, or are we just trying to say "secure channel". 

IOW, not all secure channels need to use end-to-end encryption and end-to-end encryption isn't required for all use cases (transmitting a movie ticket, for example).

Reply:
Right, this depends on our debate in #109. Perhaps for now I should remove the mention of encryption here for now?